### ISSUE TITLE
Schematics not saving when region is unrented

### ISSUE THREAD

**Issue by c0wg0d** [NLthijs48/AreaShop#66]
Using Spigot 1.7.10 and AreaShop 2.1.5.

My config has the following:

schematicProfiles:
  default:
    unrented:
      save: 'svs-%region%-%player%-%until%'
      restore: ''

I have had this setting for weeks, and yet there is not a single schem file in the plugins/AreaShop/schem folder.  Is there a separate setting that must be turned on?

**Comment by EvilOlaf** [comment]
I am using v2.2.0 and schematics working well with default config except from enabling it at default.yml. 
Did you enable them there?

**Comment by c0wg0d** [comment]
Is that this section?

general:
  schematicProfile: 'default'

If so, then yes, I have that in my default.yml.

**Comment by EvilOlaf** [comment]
Besides your `config.yml` you should have a  `default.yml`.
In this file you should edit `enableRestore: false` to `true`
However I am not sure if this file already exists in 2.1.x as I use AS since version 2.2.

**Comment by c0wg0d** [comment]
Ah, yes that option does exist and was set to false!  So I have another question.  If I set this to true, will it just ignore any save/restore lines that are blank?  I would hate to enable this and then magically my shops start disappearing because of weird restore commands I wasn't expecting.  All I want to do is save schems, not do any restoring (yet).

**Comment by EvilOlaf** [comment]
From `defaut.yml`
```
schematicProfiles:
  default:
    created:
      save: '%type%-%region%'
      restore: ''
    deleted:
      save: ''
      restore: '%type%-%region%'
    rented:
      save: ''
      restore: ''
    unrented:
      save: ''
      restore: '%type%-%region%'
    bought:
      save: ''
      restore: ''
    sold:
      save: ''
      restore: '%type%-%region%'
    resell:
      save: ''
      restore: ''
```

If you remove all `restore` settings it should just create a new schematics of areas you provide with `as add`. 
It is always handy to have a full backup before making changes like these.
`## Be carefull with these options, it could crash your server or destroy (parts of) your world if used wrong!`

**Comment by c0wg0d** [comment]
I got it working great!  Thank you for the help!  This bug/thread can be closed now.

**Comment by NLthijs48** [comment]
Good that you guys got it fixed, in AreaShop V2.2.0 there is no `enableSchematics` setting anymore in `config.yml`, to enable schematics the setting in `default.yml` can be used as you figured out already.

**Comment by EvilOlaf** [comment]
You're welcome
